[2025-02-12T23:01:14.588+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T23:01:14.603+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-06T19:14:00+00:00 [queued]>
[2025-02-12T23:01:14.611+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-06T19:14:00+00:00 [queued]>
[2025-02-12T23:01:14.611+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T23:01:14.626+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-06 19:14:00+00:00
[2025-02-12T23:01:14.633+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=26072) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T23:01:14.633+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-06T19:14:00+00:00', '--job-id', '8904', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpsxyss594']
[2025-02-12T23:01:14.634+0000] {standard_task_runner.py:105} INFO - Job 8904: Subtask fetch_and_store_crypto_data
[2025-02-12T23:01:14.634+0000] {standard_task_runner.py:72} INFO - Started process 26079 to run task
[2025-02-12T23:01:14.662+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-06T19:14:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T23:01:14.734+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-06T19:14:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-06T19:14:00+00:00'
[2025-02-12T23:01:14.734+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T23:01:14.830+0000] {logging_mixin.py:190} INFO - Données récupérées : {'Response': 'Error', 'Message': 'You are over your rate limit please upgrade your account!', 'HasWarning': False, 'Type': 99, 'RateLimit': {'calls_made': {'second': 11, 'minute': 36, 'hour': 174, 'day': 8379, 'month': 8379, 'total_calls': 8379}, 'max_calls': {'second': 20, 'minute': 300, 'hour': 3000, 'day': 7500, 'month': 11000}}, 'Data': {}}
[2025-02-12T23:01:14.836+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans PostgreSQL : string indices must be integers, not 'str'
[2025-02-12T23:01:14.855+0000] {logging_mixin.py:190} INFO - Données uploadées dans MinIO.
[2025-02-12T23:01:14.856+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T23:01:14.869+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T23:01:14.870+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-06T19:14:00+00:00, execution_date=20231006T191400, start_date=20250212T230114, end_date=20250212T230114
[2025-02-12T23:01:14.928+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T23:01:14.959+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T23:01:14.962+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
